src/index.html

```diff
@@ -7,8 +7,49 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo--img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav--link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav--link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav--link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#laptops"
+              data-qa="hover"
+              class="nav--link"
+            >Laptops & Computers</a>
+          </li>
+          <li class="nav__item">
+            <a href="#gadgets" class="nav--link">Gadgets</a>
```

Comment:
```suggestion
            <a href="#gadgets" class="nav__link">Gadgets</a>
```
As that is not a modifier, that is element

Reply:
+